core/src/main/scala/kafka/server/ControllerApis.scala

```diff
@@ -1065,7 +1065,8 @@ class ControllerApis(
     }
     // Unlike on the broker, DESCRIBE_CLUSTER on the controller requires a high level of
     // permissions (ALTER on CLUSTER).
-    authHelper.authorizeClusterOperation(request, ALTER)
```

Comment:
Unrelated to this PR but probably good to have. Could we update `authorizeClusterOperation` to log the missing permission as well in the exception.

Reply:
Thanks for review 😄.